sjb/actions/generate_artifacts.py

```diff
@@ -10,7 +10,7 @@
 ARTIFACT_DIR="$( pwd )/artifacts/generated"
 rm -rf "${ARTIFACT_DIR}"
 mkdir "${ARTIFACT_DIR}"
-{%- for name, action in artifacts.iteritems() %}
+{%- for name, action in artifacts.items() %}
```

Comment:
python3 has no iteritems for dictionaries.  items is backwards compatible, but may be marginally slower for large dictionaries.

Reply:
Can you split this and the resulting ordering changes into a separate PR please?